package.json

```diff
@@ -28,22 +34,28 @@
   },
   "homepage": "https://github.com/SortableJS/angular-sortablejs#readme",
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/forms": "^2.0.0",
-    "rxjs": "^5.0.0-beta.12",
+    "@angular/common": "2.1.0",
+    "@angular/compiler": "2.1.0",
+    "@angular/compiler-cli": "2.1.0",
+    "@angular/core": "2.1.0",
+    "@angular/forms": "2.1.0",
+    "rxjs": "5.0.2",
     "sortablejs": "^1.4.2",
-    "zone.js": "^0.6.17"
+    "zone.js": "0.6.17"
   },
   "devDependencies": {
     "@types/core-js": "^0.9.32",
     "@types/node": "^6.0.38",
     "@types/sortablejs": "^1.3.29",
     "@types/systemjs": "^0.19.30",
     "git-validate": "^2.1.4",
+    "rimraf": "^2.5.2",
     "tslint": "^3.13.0",
-    "typescript": "^2.0.0"
+    "typescript": "2.0.3"
```

Comment:
As per previous comments, if you revert to `^2.0.0`, as of now you will definitely download 2.1.4 actually, and that will cause you troubles. You might consider turning to the less-relaxed tilde range: `~2.0.0`